### PR TITLE
Payec 249 print temp

### PR DESCRIPTION
**Pull request by YuanZHANG910**
PAYEC-249: this is a temporary work for print, this will show all tabs as default, then let users to chose which page or all pages they want to print at print GUI.

### CODE REVIEW

File: app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala [hunk 1]
```diff
@@ -148,11 +149,17 @@ object TaxResult {
...
 
-  def getPeriodFromHtml(tab: String): String = {
-    tab match {
-      case "tabContentmonthly" => ???
-      case "tabContentweekly" => ???
-      case _ => ???
+  def getPeriodFromHtml(tab: scala.xml.Elem): String = {
+    println("")
+    println("")
+    println(tab)
```
Comment (JTY009) [inline app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala]: Do we need the println?

File: app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala [hunk 2]
```diff
@@ -148,11 +149,17 @@ object TaxResult {
...
+    println("")
+    println("")
+    tab.toString() match {
+      case "tabContentmonthly" => ""
+      case "tabContentweekly" => ""
+      case "tabContentmonthly" => "111"
```
Comment (JTY009) [inline app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala]: Why 111?

File: app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala [hunk 3]
```diff
@@ -148,11 +149,17 @@ object TaxResult {
...
+      case _ => "222"
```
Comment (JTY009) [inline app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala]: Why 222?

Comment (YuanZHANG910) [inline app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala]: forgot to delete them, they were there for tests.

Comment (JTY009) [inline app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala]: Ah thought so but just checking

**Pull request merged**